config.md

```diff
@@ -124,6 +124,10 @@ Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being a
      If there are no labels then this property MAY either be absent or an empty map.
      Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown labels key.
 
+  - **StopSignal** *string*, OPTIONAL
+
+    The field contains the system call signal that will be sent to the container to exit. The signal can be a valid unsigned number that matches a position in the kernel’s syscall table, for instance `9`, or a signal name in the format `SIGNAME`, for instance `SIGKILL`.
```

Comment:
Can we require the named version (e.g. `SIGKILL`) and not allow numbers (e.g. `9`)?  A fair number of names are [portable across POSIX systems][1] (and [a handful are required by ISO C][1]), but POSIX has very little to say about the semantics for a particular signal number (e.g. it does not require `SIGKILL` to be 9).  And the Linux man pages [has][2]:

> Because the range of available real-time signals varies according to the glibc threading implementation… programs should *never refer to real-time signals using hard-coded numbers*, but instead should always refer to real-time signals using the notation `SIGRTMIN+n`…

And [more generally][3]

> Several signal numbers are architecture-dependent, as indicated in the "Value" column.

So using numbers is less portable for no obvious benefit (or maybe there is a benefit, and I'm just not seeing it?).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html
[2]: https://git.kernel.org/cgit/docs/man-pages/man-pages.git/tree/man7/signal.7?h=man-pages-4.09#n371
[3]: https://git.kernel.org/cgit/docs/man-pages/man-pages.git/tree/man7/signal.7?h=man-pages-4.09#n224